internal/orchestrator/orchestrator.go

```diff
@@ -393,7 +369,6 @@ func (or *orchestrator) initDataExchange(ctx context.Context) (err error) {
 	fb := database.IdentityQueryFactory.NewFilter(ctx)
 	nodes, _, err := or.database.GetIdentities(ctx, fb.And(
```

Comment:
I think I'm right in saying, this section will move to being one-per-namespace when @shorsher has completed the work to make DX plugins numerous.... so the namespace in the query query will come back at that point.

Reply:
Yes. For now data exchange is still global, so it needs to know about all the nodes across namespaces - but when the plugin moves to a namespace, it will only deal with the nodes in its own namespace.